ocfweb/stats/printing.py

```diff
@@ -125,7 +123,10 @@ def _toner_used_by_printer(printer, cutoff=.05, since=date(2017, 8, 20)):
             ABS(pct_diff)<%s
         ''', (cutoff),
         )
-        return float(cursor.fetchone()['toner_used'])
+        result = cursor.fetchone()['toner_used']
+        # The none check is here as a kind of failsafe for if the request is run on a printer
+        # that is no longer in use, although this should no longer happen in the first place
+        return [None if result is None else float(result)]
```

Comment:
If `result` should never be None, then add an assert statement and get rid of the comment.

Reply:
Is it ok to leave the result in a singleton list like this? I left it like that since the HTML on the stats page loops through a list of toner changes to generate rows, and even though I could've changed that, I didn't want to in case we have the table display toner changes for multiple semesters/different time periods in the future.

Reply:
personally i'd drop the list, usually when we try to be defensive and leave code around in case somebody wants to use it later, it just ends up being technical debt.

if people want to find the old code, they can also grab it from the git history.